### DISCUSSION TITLE
I keep seeing RPC errors when editing code...

### DISCUSSION THREAD

**Discussion by augustocdias** [mrcjkb/rustaceanvim#413] [Q&A] [answered]
I keep seeing this error when editing code:

```
lsp_signatur handler RPC[Error] code_name = ContentModified, message = "content modified"
```

Is this some config I'm doing wrong? What is the cause for that? I've searched in rust analyzer issues and the only thing I found was someone using an older version with a newer lsp client which is not my case. I'm using latest of everything...

My rustacean config:

https://github.com/augustocdias/dotfiles/blob/main/.config/nvim/lua/setup/lsp/rust.lua#L10

**Comment by mrcjkb** [comment]
Hey 👋 

I suggest you follow the readme's troubleshooting guide, starting with the provided minimal config template. 
If you can't reproduce it with the minimal config, you can bisect your config and plugins until you find what is causing the error.

**Reply by augustocdias** [reply, marked as answer]
After a lot of search I found out the problem was in lsp-signature plugin instead... I've managed to solve it by loading it only after the lsp attaches to a buffer.
